docs/general/ddl/shard-allocation.rst

```diff
@@ -38,6 +38,12 @@ defined by altering a table) from one set of nodes to another:
    Assign the table to a node whose *{attribute}* has **none** of the
    comma-separated values.
 
+.. NOTE::
+
+    These settings are not mutually exclusive. We can for instance have a
```

Comment:
I think we typically don't use "we" in the documentation, but rather address the user as "you".